src/targets/gpu/prefuse_ops.cpp

```diff
@@ -192,8 +179,7 @@ struct find_gemm_softmax_gemm
     auto matcher() const
     {
         auto gemm1 = match::skip(match::name("contiguous"))(match::name("dot")(
-            match::any_of(is_ck_gemm(), is_mlir_gemm(), is_test_gemm(enable_attention))
-                .bind("gemm1")));
+            match::any_of(is_ck_gemm(), is_test_gemm(enable_attention)).bind("gemm1")));
```

Comment:
Im not sure what this means, can you elaborate? 
`is_test_gemm` is used only for the test suite so that these matchers get tested regardless of which environment variables are set

Reply:
I just meant that the naming is odd (to me at least). I could understand if it were `is_test` or `is_gemm` or something like that. But this isn't a big deal, just call it `is_nitpicking()`  :-)